unscrobbler.js

```diff
@@ -177,8 +177,8 @@ function validatePage() {
   const url = new URL(document.URL);
 
   if (!username) alert('You are not logged in.\nPlease log in to use Last.FM Unscrobbler.');
-  else if (url.pathname.endsWith(username) || url.pathname.endsWith(`${username}/library`)) unscrobbler();
-  else alert('Last.FM Unscrobbler works only on:\n· Your profile page\n· Your library scrobbles page');
+  else if (url.pathname.endsWith(username) || url.pathname.endsWith(`${username}/library`) || (url.pathname.includes(`${username}/library`) && url.pathname.split('/').slice(-2)[0] == "_")) unscrobbler();
```

Comment:
Nice!! 
I think it'd be good to space out the condition for readability, and also to add a comment to clear out what's the condition is for :) 

```suggestion
  else if (
    url.pathname.endsWith(username) ||
    url.pathname.endsWith(`${username}/library`) ||
    // Covers the library track page
    // The path includes "username/library" and between the album and the track there is an underscore ("album/_/track").
    (url.pathname.includes(`${username}/library`) && url.pathname.split('/').slice(-2)[0] == '_')
  )
```